### PR TITLE
OHRM5X-1513: Performance evaluation form UI enhnacements

### PR DESCRIPTION
**Pull request by Super-Chama**

### CODE REVIEW

**Comment by Super-Chama** [comment]
**Web UI Before**

os51al22-os-guardian orangehrm com_web_index php_performance_reviewEvaluateByAdmin_7

**Web UI After**

php74_orangehrm5_web_index php_performance_reviewEvaluateByAdmin_1

**Comment by Super-Chama** [comment]
**Mobile UI Before**
os51al22-os-guardian orangehrm com_web_index php_performance_reviewEvaluateByAdmin_7 (1)

**Mobile UI After**
php74_orangehrm5_web_index php_performance_reviewEvaluateByAdmin_1 (1)

File: src/client/src/orangehrmPerformancePlugin/components/review-summary.scss [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    margin-bottom: 0.5rem;
+  }
+
+  &-details {
+    display: flex;
+  }
+
+  &-details-grid {
+    width: 100%;
+  }
```
Comment (devishke-orange) [inline src/client/src/orangehrmPerformancePlugin/components/review-summary.scss:91]: Maybe can add `align-items: center` here

Before:
image

After:
image

**Review by devishke-orange — APPROVED** [review]
Looks great :+1: . Just had one suggestion.

File: src/client/src/orangehrmPerformancePlugin/components/review-summary.scss (hunk 1 above)
Comment (Super-Chama) [reply src/client/src/orangehrmPerformancePlugin/components/review-summary.scss]: Done

**Review by RajithaKumara — APPROVED** [review]

File: src/test/functional/cypress/integration/feature/admin/empStatus.spec.js [hunk 2]
```diff
@@ -102,7 +102,10 @@ describe('Admin - Employment Status', function () {
     it('update emp status', function () {
       cy.task('db:restore', {name: 'empStatus'});
       cy.loginTo(this.user, '/admin/employmentStatus');
-      cy.get(':nth-child(2) > .oxd-icon').click();
+      cy.wait('@getEmpStatus');
+      cy.get(
+        '.oxd-table-body > :nth-child(1) .oxd-table-cell-actions > :nth-child(2)',
```
Comment (Super-Chama) [inline src/test/functional/cypress/integration/feature/admin/empStatus.spec.js:107]: `.oxd-table-body > :nth-child(1)` - First row of cardtable
`.oxd-table-cell-actions > :nth-child(2)` - 2nd action of card table (edit)

**Pull request merged**